packages/wdio-applitools-service/src/index.ts

```diff
@@ -8,14 +8,6 @@ const DEFAULT_VIEWPORT = {
     height: 900
 }
 
-declare global {
```

Comment:
This is not needed after upgrading `@types/node` to `^12`, typings for `globalThis` will support extending global object